capstone/src/test/java/com/google/sps/AssociationAnalysisTest.java

```diff
@@ -0,0 +1,104 @@
+package com.google.sps.servlet;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import com.google.sps.AssociationAnalysis;
+import com.google.sps.AssociationResult;
+import com.google.sps.EntitySentiment;
+import java.lang.NullPointerException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class AssociationAnalysisTest extends Mockito{
+
+  @Test
+  public void testEmpty() throws Exception {
+    AssociationAnalysis analysis = new AssociationAnalysis();
+    ArrayList<AssociationResult> res = analysis.calculateScores(new ArrayList<EntitySentiment>());
+    assertEquals(res, new ArrayList<AssociationResult>());
+  }
+
+  @Test
+  public void allDistinct() {
```

Comment:
Always prefix each test with "test"

So in this case `testAllDistinct`

Reply:
Fixed!